CHANGELOG

```diff
@@ -1,3 +1,7 @@
+3.1.0 2021-01-25
+
+Follow-up /usr/share/zorp directory structure change
```

Comment:
Please indicate that this plugin should be used from 6.9 and the older versions should configure the dictionary path